### ISSUE TITLE
How to set `eos-on-timestamps-reset=False` in module.yml

### ISSUE THREAD

**Issue by nikcleju** [insight-platform/Savant#681]
I have a bad RTSP stream which sometimes provides non-monotonous timestamps, and it's causing frequent source resets (every few seconds), with messages like "'Timestamps on source xxx non-monotonous. Resetting source."

Hacking around I've found the `eos-on-timestamps-reset` property in `savant_rs_video_demux.py`, which is set to True by default. If i hack the script to turn it False, it stops the resets.

Is there a clean way to set this property from `module.yml`? I've adapted one of samples in the samples folder, using a RTSP Source adapter, so I don't have a `source:` element in my `pipeline:`.

Is there a  downside to setting `eos-on-timestamps-reset` to False?

**Comment by bwsw** [comment]
Hello @nikcleju ,
I would recommend first trying fixing PTS with the adapter, e.g. you can utilize ffmpeg source adapter and pass extra arguments to get rid of PTS, like `fflags +genpts`, not sure it helps. 

But, in general, I assume it is better to do this sort of fix outside of the module, in the adapter. E.g. you can write a buffer adapter that just reads from the stream, fixes pts, and writes s to the stream. You can find a sample of such an adapter here:

https://github.com/insight-platform/Savant/blob/develop/adapters/python/bridge/buffer.py

it is more complex than you need, but you can completely remove unnecessary gears and use it just to fix PTS.

**Comment by nikcleju** [comment]
Thank you, I will look into the adapters, but I don't really know how to fix the PTS myself. When I run one of the original DeepStream samples on the same RTSP stream, things just work and the output video is completely smooth. I haven't even realized there might be a timestamp problem in the RTSP stream until I switched to Savant. 

I already use ffmpeg source adapter, and I tried various parameter combinations (`use_wallclock_as_timestamps=1`,`fps_mode=1`,`fflags=+genpts+igndts+ignidx`), but haven't seen an improvement. It's either still resetting, or the output video is non-smooth (when I `use_wallclock_as_timestamps`).  I don't know how the DeepStream samples handle this to get a smooth output.

In my experiments I haven't seen any problem yet caused by setting `eos-on-timestamps-reset=False`. But I'm still scratching the surface of Savant.
